### ISSUE TITLE
Incrementation bug

### ISSUE THREAD

**Issue by cdgls** [jamievicary/globular#40]
When making a change in a unprojected view (ie not the highest projection), the displayed picture is not incrementing to the result of the change you made, so it looks as though nothing is happening.  (Ie a new slice is created, but the picture doesn't display the newly created slice.)

**Comment by jamievicary** [comment]
Yeah, I agree this is really annoying, thanks for listing it as a bug.

**Comment by cdgls** [comment]
Just to be clear, it is incrementing the first slice number, but not the second, and that's the bug --- it should increment the second as well.

**Comment by jamievicary** [comment]
I think that whatever slice or projection you're in, after making a move
with the mouse, you should be shown the slice in which the 'ordinary
effect' of that move is visible - do you agree?

**Comment by cdgls** [comment]
I think I agree, though I'm not sure what distinction is being made by 'ordinary effect' --- do you mean the effect that would be visible at that slice/projection currently on view?

**Comment by jamievicary** [comment]
I mean the effect you would see if the diagram you were looking at was
completely unsliced (ie if you clicked restrict before doing the move.)

**Comment by cdgls** [comment]
Then yes, I agree.
